pysatMissions/instruments/pysat_sgp4.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+Produces satellite orbit data. Orbit is simulated using
+Two Line Elements (TLEs) and SGP4.
+
+"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+import functools
+import os
+
+import pandas as pds
+import pysat
+
+from pysatMissions.instruments import _core as mcore
+
+# pysat required parameters
+platform = 'pysat'
+name = 'sgp4'
+# dictionary of data 'tags' and corresponding description
+tags = {'': 'Satellite simulation data set'}
+# dictionary of satellite IDs, list of corresponding tags
+sat_ids = {'': ['']}
+test_dates = {'': {'': pysat.datetime(2018, 1, 1)}}
+
+
+def init(self):
+    """
+    Adds custom calculations to orbit simulation.
+    This routine is run once, and only once, upon instantiation.
+
+    """
+
+    pass
+
+
+def load(fnames, tag=None, sat_id=None, obs_long=0., obs_lat=0., obs_alt=0.,
+         TLE1=None, TLE2=None):
+    """
+    Returns data and metadata in the format required by pysat. Finds position
```

Comment:
'Finds position' is confusing to me. Perhaps Models position, or Generates position....